src/application/service/evaluation.go

```diff
@@ -214,25 +218,32 @@ func (e *evaluationService) EvaluateRun(src, name string, id uuid.UUID, inputs m
 			} else {
 				for _, tg := range job.TaskGroups {
 					for _, t := range tg.Tasks {
-						_, found := t.Config["console"]
-						if !found {
-							t.Config["console"] = "pipe"
-						}
-
-						if pkgsI, found := t.Config["packages"]; found {
-							keys := map[string]bool{}
-							for _, pkg := range pkgsI.([]interface{}) {
-								keys[pkg.(string)] = true
+						switch t.Driver {
+						case "podman":
+							if _, found := t.Config["packages"]; found {
+								delete(t.Config, "packages")
 							}
```

Comment:
What is this needed for?

Reply:
the `packages` field is always set by https://github.com/input-output-hk/cicero/blob/1c6ccd20fd6c7d849b738b1d98200d9e6218580d/jobs/cicero.cue#L185-L190 and that field is only valid when the driver is `nix`. With podman that results in being unable to schedule the job.

Reply:
I would consider that a bug in the transformer so we should fix it there instead